docs/codespace_pr_reviews.md

```diff
@@ -0,0 +1,29 @@
+# Using Codesapces to review PRs without pulling a repo to your machine
+
+1. go to the PR
+2. click the `Code` button in the top right corner
+3. click the `Codespaces` tab
+4. `Create new codespace`
+
+   ![Create a new codespace screenshot](https://github.com/trussworks/react-uswds/assets/59394696/9ebfff8a-353c-4064-bf70-8712bbfbbeda)
+
+5. **Wait** for the codespace to finish setting up, you should see stuff happening in the `Terminal`. If you mess with it before it is done it might not work. If you mess this up, you might see an error about your node version.
+6. if the yarn install in that process was not successful, run `yarn install`. Wait until it is done before you do anything. takes ~5ish minutes
+7. if it is successful run `yarn storybook`
```

Comment:
Running it this first time, mine got stuck on 99% done:
<img width="698" alt="image" src="https://github.com/trussworks/react-uswds/assets/13249580/b9d66c78-3659-4c18-a7b4-c6d493354135">

Reply:
re conversation on Slack- Yes, adding the screenshot of the port and pointing to the globe as the link would be helpful.